components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -390,6 +397,7 @@ type ApiResult struct {
 	Check   string       `json:"check"`
 	Checks  []Checks     `json:"checks"`
 	Error   *fiber.Error `json:"error,omitempty"`
+	Skipped bool         `json:"skipped"`
```

Comment:
Do we need this skip here?

Reply:
Yes we do need this, I confirmed with @Dmaddu